Description: Fixes IMAGE_TYPES
 By default, IMAGE_TYPES is empty. This patch sets the Debian default to "png
 gif".
Author: Roland Stigge <stigge@antcom.de>
Bug-Debian: http://bugs.debian.org/540197

--- latex2html-2008-debian1.orig/docs/.latex2html-init
+++ latex2html-2008-debian1/docs/.latex2html-init
@@ -5,6 +5,7 @@ $INDEX_WIDTH  = 250;
 $CONTENTS_WIDTH  = 250;
 $HTML_VALIDATE = '';
 $LOCAL_ICONS = 1;
-
+@IMAGE_TYPES = qw(png gif);
+$IMAGE_TYPE  = $IMAGE_TYPES[0];
 
 1;

